packages/lib-classifier/src/plugins/tasks/survey/components/components/Chooser/components/Choices/components/ChoiceButton.js

```diff
@@ -5,113 +5,157 @@ import {
-
-    img {
-      margin: ${props => props.columnsCount === 1 ? '0 1ch 0 0' : '0'};
-    }
+const StyledButton = styled(Button)`
+  flex-grow: 1;
+  height: 100%;
+`
 
-    span {
-      text-align: ${props => props.columnsCount === 1 ? 'left' : 'center'};
-    }
-  }
+const StyledImage = styled(Image)`
+  border-radius: 4px;
+  object-fit: cover;
 `
 
+const DEFAULT_HANDLER = () => true
+
 function ChoiceButton({
-  ariaChecked = undefined,
   choiceId = '',
   choiceLabel = '',
-  columnsCount = 1,
   disabled = false,
   hasFocus = false,
-  onChoose = () => true,
-  onKeyDown = () => true,
-  role='button',
+  onChoose = DEFAULT_HANDLER,
+  onDelete = DEFAULT_HANDLER,
+  onKeyDown = DEFAULT_HANDLER,
   selected = false,
+  shadedBackground = false,
   src = '',
   tabIndex = -1,
   thumbnailSize = 'none'
 }) {
-  const choiceButton = useRef(null)
+  const choiceMenuItem = useRef(null)
   const handleClick = useCallback(() => {
     onChoose(choiceId)
   }, [choiceId, onChoose])
+  const handleDelete = useCallback(() => {
+    onDelete(choiceId)
+  }, [choiceId, onDelete])
   const handleKeyDown = useCallback((event) => {
     onKeyDown(choiceId, event)
   }, [choiceId, onKeyDown])
 
   useEffect(() => {
-    if (choiceButton && hasFocus) {
-      choiceButton.current.focus()
+    if (choiceMenuItem && hasFocus) {
+      choiceMenuItem.current.focus()
     }
   })
 
-  let thumbnailHeight = 0
-  if (thumbnailSize === 'small') {
-    thumbnailHeight = 21
-  }
-  if (thumbnailSize === 'medium') {
-    thumbnailHeight = 42
-  }
-  if (thumbnailSize === 'large') {
-    thumbnailHeight = 84
-  }
-
+  const background = selected
+    ? 'neutral-1'
+    : shadedBackground
+      ? {
+          dark: 'dark-4',
+          light: 'light-1'
+        }
+      : {
+          dark: 'dark-5',
+          light: 'neutral-6'
+        }
+  const border = selected
+    ? { color: 'neutral-6', size: '1px' }
+    : undefined
+  const textColor = selected 
+    ? 'neutral-6' 
+    : {
+        dark: 'neutral-6',
+        light: 'neutral-7'
+      }
+  const thumbnailHeight = 150
   const thumbnailWidth = Math.round(thumbnailHeight * THUMBNAIL_ASPECT_RATIO)
   const thumbnailSrc = `https://thumbnails.zooniverse.org/${thumbnailWidth}x${thumbnailHeight}/${src.slice(8)}`
   
   return (
-    <Button
-      ref={choiceButton}
-      aria-checked={ariaChecked}
-      disabled={disabled}
+    <StyledBox
+      ref={choiceMenuItem}  
+      // TODO: add the following to translations
+      a11yTitle={`${choiceLabel}` + (selected ? '; identified' : '')}
+      aria-haspopup='true'
+      role='menuitem'
+      align='center'
+      background={background}
+      border={border}
+      direction='row'
       fill
-      label={
-        <StyledBox
-          align='center'
-          columnsCount={columnsCount}
-          fill
-          forwardedAs='span'
+      onKeyDown={disabled ? DEFAULT_HANDLER : handleKeyDown}
```

Comment:
This is triggered when I key `enter` with the delete button focused, opening the Choice component, when it should delete the choice (identification and related answers) if I key `enter` with the delete choice button focused.